src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -191,34 +197,32 @@ private async Task InitializeColorsAsync(CancellationToken cancellationToken)
 
         protected override async Task LoadComponentsAsync(CancellationToken cancellationToken)
         {
-            await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
+            await TaskScheduler.Default;
 
-            await GetServiceAsync(typeof(SVsTaskStatusCenterService)).ConfigureAwait(true);
-            await GetServiceAsync(typeof(SVsErrorList)).ConfigureAwait(true);
-            await GetServiceAsync(typeof(SVsSolution)).ConfigureAwait(true);
-            await GetServiceAsync(typeof(SVsShell)).ConfigureAwait(true);
-            await GetServiceAsync(typeof(SVsRunningDocumentTable)).ConfigureAwait(true);
-            await GetServiceAsync(typeof(SVsTextManager)).ConfigureAwait(true);
+            await GetServiceAsync(typeof(SVsTaskStatusCenterService)).ConfigureAwait(false);
+            await GetServiceAsync(typeof(SVsErrorList)).ConfigureAwait(false);
+            await GetServiceAsync(typeof(SVsSolution)).ConfigureAwait(false);
+            await GetServiceAsync(typeof(SVsShell)).ConfigureAwait(false);
+            await GetServiceAsync(typeof(SVsRunningDocumentTable)).ConfigureAwait(false);
+            await GetServiceAsync(typeof(SVsTextManager)).ConfigureAwait(false);
```

Comment:
IIRC this was being done so future blocking GetService() calls were cheap, do we still need them after this change?